src/main/js/containers/content/generator/CourseTable.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+import CourseTable from '../../../components/CourseTable';
+
+const mapStateToProps = (state, ownProps) => ({
+  ...ownProps,
```

Comment:
Don't need to `...ownProps`. The props in this object are passed to the props of the actual object.

Reply:
Removed.